AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java

```diff
@@ -1448,6 +1451,20 @@ public boolean shouldOverrideUrlLoading(WebView view, WebResourceRequest request
                 return filterUrl(url);
             }
 
+            @Override
+            @TargetApi(Build.VERSION_CODES.N)
+            public WebResourceResponse shouldInterceptRequest(WebView view, WebResourceRequest request) {
+                WebResourceResponse webResourceResponse=null;
+                if(AdaptionUtil.hasReducedPreferences()) {
```

Comment:
I believe you want to key of web browser availability here, not reduced preferences?
```suggestion
                if (AdaptionUtil.hasWebBrowser()) {
```